sympy/stats/tests/test_continuous_rv.py

```diff
@@ -189,8 +192,12 @@ def test_benini():
     alpha = Symbol("alpha", positive=True)
     beta = Symbol("beta", positive=True)
     sigma = Symbol("sigma", positive=True)
-
+    x = Symbol('x')
```

Comment:
These are not needed since they are globally defined. (Or am I missing something?)

Reply:
A few of the other tests also redefined the variable. 
So, I did it to be safe, just in case.

I will try testing without redefining x.
If that works, should I also remove rest of the excess definitions too?